modin/core/storage_formats/base/doc_utils.py

```diff
@@ -321,7 +321,7 @@ def doc_reduce_agg(method, refer_to, params=None, extra_params=None):
 
         {build_rules}
     """,
-    refer_to_module_name="DataFrame.Resampler",
+    refer_to_module_name="Resampler",
```

Comment:
```suggestion
    refer_to_module_name="resample.Resampler",
```
For the actual path to the method:
```python
>>> # Before applying the suggestion
>>> print(BaseQueryCompiler.resample_nunique.__doc__)
...
Notes
-----
Please refer to ``modin.pandas.Resampler.nunique`` for more information
about parameters and output format.
>>>
>>> # After applying the suggestion
>>> print(BaseQueryCompiler.resample_nunique.__doc__)
Notes
-----
Please refer to ``modin.pandas.resample.Resampler.nunique`` for more information
about parameters and output format.

```

Reply:
Done, thank you.